pkg/bosh/manifest/containerization/containerization.go

```diff
@@ -1,13 +1,55 @@
-package manifest
+// Package containerization loads the kubernetes specifics parts, like
+// BOSHContainerization, from the BOSH manifest.
+package containerization
 
 import (
+	yaml "gopkg.in/yaml.v2"
+
 	corev1 "k8s.io/api/core/v1"
 
 	"code.cloudfoundry.org/cf-operator/pkg/bosh/bpm"
 )
 
+// Manifest is a BOSH deployment manifest
+type Manifest struct {
```

Comment:
Why these structs like Manifest, InstanceGroup should be containerization-specified? I think it should be bosh part.

Reply:
I thought I need those structs so we can get to the affinity and bosh_containerization keys in the tree.